src/epstats/prometheus.py

```diff
@@ -5,9 +5,11 @@
 application_prefix = "ep_stats_"
 
 
-def get_prometheus_metric(metric_name: str, metric_type: type) -> Union[Counter, Summary]:
+def get_prometheus_metric(metric_name: str, metric_type: type, labels: list[str] = None) -> Union[Counter, Summary]:
+    if labels is None:
+        labels = []
```

Comment:
Maybe we can simplify this?
```suggestion
def get_prometheus_metric(metric_name: str, metric_type: type, labels: list[str] = []) -> Union[Counter, Summary]:
```

Reply:
ohh good point 👍  nevermind then